### PR TITLE
Inmobi: user sync

### PR DESCRIPTION
**Pull request by armon823**

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -611,6 +611,7 @@ func (cfg *Configuration) setDerivedDefaults() {
...
 	setDefaultUsersync(cfg.Adapters, openrtb_ext.BidderGumGum, "https://rtb.gumgum.com/usync/prbds2s?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&r="+url.QueryEscape(externalURL)+"%2Fsetuid%3Fbidder%3Dgumgum%26gdpr%3D{{.GDPR}}%26gdpr_consent%3D{{.GDPRConsent}}%26uid%3D")
 	setDefaultUsersync(cfg.Adapters, openrtb_ext.BidderImprovedigital, "https://ad.360yield.com/server_match?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&r="+url.QueryEscape(externalURL)+"%2Fsetuid%3Fbidder%3Dimprovedigital%26gdpr%3D{{.GDPR}}%26gdpr_consent%3D{{.GDPRConsent}}%26uid%3D%7BPUB_USER_ID%7D")
+	setDefaultUsersync(cfg.Adapters, openrtb_ext.BidderInMobi, "https://id5-sync.com/i/495/0.gif?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&callback="+url.QueryEscape(externalURL)+"%2Fsetuid%3Fbidder%3Dinmobi%26gdpr%3D{{.GDPR}}%26gdpr_consent%3D{{.GDPRConsent}}%26uid%3D%7BID5UID%7D")
```
Comment (SyntaxNode) [inline config/config.go:614]: Please confirm the `495` or `0.gif` identifiers do not refer to any specific host of Prebid Server, such as AppNexus.

**Comment by SyntaxNode** [comment]
While I've got you here, could you please update the support email adress for Inmobi? The last time I tried to send an email to that address I received the following error from your email server:

```
550 5.7.193 UnifiedGroupAgent; Delivery failed because the sender isn't a group member or external senders aren't permitted to send to this group.
```

**Comment by armon823** [comment]
@SyntaxNode sure I can provide that. What will be the expected frequency of emails to the email address? What kind of support?

File: config/config.go (hunk 1 above)
Comment (armon823) [reply config/config.go:614]: these are part of the confirmed url and should be correct

Comment (SyntaxNode) [reply config/config.go:614]: What do these values refer to? I need to be sure they don't refer to a specific host.

**Comment by SyntaxNode** [comment]
> @SyntaxNode sure I can provide that. What will be the expected frequency of emails to the email address? What kind of support?

Great. Thank you.

The support address has been provided by Inmobi as `prebid-support@inmobi.com` in the file `inmobi.yaml`. It is used by us to reach out with adapter maintenance requests and by publishers with configuration issues.

File: config/config.go (hunk 1 above)
Comment (armon823) [reply config/config.go:614]: This is the url that was provided to us by the vendor. I confirmed that it does not point to any specific prebid server hosts.

Question: Will the user sync process work with any hosted Prebid servers?

**Review by VeronikaSolovei9 — APPROVED** [review]
LGTM

**Comment by armon823** [comment]
@VeronikaSolovei9 Great. Can we merge?

**Review by SyntaxNode — APPROVED** [review]

**Comment by armon823** [comment]
@SyntaxNode  @VeronikaSolovei9 Can I please know when the next release will be?

**Pull request merged**